src/main/resources/log4j2.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Configuration status="INFO">
+    <Appenders>
+        <Console name="Console" target="SYSTEM_OUT">
+            <PatternLayout pattern="%d{HH:mm:ss.SSS} [%t] %-5level %logger{36} - %msg%n"/>
+        </Console>
+        <File name="File" fileName="logs/app.log">
+            <PatternLayout>
+                <Pattern>%d %p %c: %line %msg%n</Pattern>
```

Comment:
you can use here %L instead of %line, just in case, not necessary
```suggestion
                <Pattern>%d %p %c:%L %msg%n</Pattern>
```

Reply:
Thanks for your suggestion but I made it on purpose) 
In my opinion, this option is significantly more informative 😊

Reply:
I'd also add a level and date and time formatted